packages/customize-widgets/src/components/inserter/style.scss

```diff
@@ -0,0 +1,60 @@
+$customize-widgets-header-width: 50px;
+$inserter-transition-ms: 150;
+
+.customize-widgets-layout__inserter-panel-backdrop {
+	background: rgba(0, 0, 0, 0.5);
+	position: absolute;
+	height: 100%;
+	width: 100%;
+	opacity: 0;
+	transition: opacity #{$inserter-transition-ms}ms ease-in-out;
+
+	&[data-enter] {
+		opacity: 1;
+	}
+}
+
+.customize-widgets-layout__inserter-panel {
+	background: #fff;
+	position: absolute;
+	height: 100%;
+	transform: translateX(-100%);
+	transition: transform #{$inserter-transition-ms}ms ease-in-out;
+
+	&[data-enter] {
+		transform: translateX(0);
```

Comment:
If you add `outer-section-open` to the `body` element's `class`, you'll see that the iframe will dim and that `#customize-sidebar-outer-content` will slide out. Maybe we could reuse that instead of adding our own slide-out and fade animation?

Reply:
Could borrow what https://github.com/WordPress/gutenberg/pull/29755 does to make it so that the inspector appears in `#customize-theme-controls` to make it so that the inserter appears in `#customize-sidebar-outer-content`.